runtime/parachains/src/configuration.rs

```diff
@@ -193,6 +193,7 @@ pub struct HostConfiguration<BlockNumber> {
 	/// How long after dispute conclusion to accept statements.
 	pub dispute_post_conclusion_acceptance_period: BlockNumber,
 	/// The maximum number of dispute spam slots
+	/// TODO: Should be removed in next version
```

Comment:
I read the comment explaining that changes to this struct are handled by migrations. 

Are we even allowed to remove fields?

Reply:
Yes, since it's only used by relay chain and not exposed in `AbridgedHostConfiguration`.

An example of removing a field with migration is #6271.

Reply:
To avoid conflicts with #6271 I'll remove the field in a separate PR.

Reply:
#6271 already has got conflicts with the `HostConfiguration` migration so I'll push my changes. Hopefully everything can go in v4.